diff -ru a/Source/CursesDialog/CMakeLists.txt b/Source/CursesDialog/CMakeLists.txt
--- a/Source/CursesDialog/CMakeLists.txt	2012-11-28 00:26:32.000000000 +1100
+++ b/Source/CursesDialog/CMakeLists.txt	2013-01-01 23:42:59.813826780 +1100
@@ -33,5 +33,5 @@
 add_executable(ccmake ${CURSES_SRCS} )
 target_link_libraries(ccmake CMakeLib)
-target_link_libraries(ccmake cmForm)
+target_link_libraries(ccmake cmForm tinfo)
 
 install_targets(/bin ccmake)
